larpandora/LArPandoraInterface/StandardPandora_module.cc

```diff
@@ -63,6 +63,9 @@ DEFINE_ART_MODULE(StandardPandora)
 #include "larpandoracontent/LArPlugins/LArPseudoLayerPlugin.h"
 #include "larpandoracontent/LArPlugins/LArRotationalTransformationPlugin.h"
 
+#ifdef LIBTORCH_DL
+    #include "larpandoradlcontent/LArDLContent.h"
```

Comment:
Ah, I wondered when I looked at the LArReco PR, whether you'd like to format the preprocessor in that manner there too (as it stands, it's different in the two use-cases).

Reply:
Yes, I think I changed it in the other case because there was already an example using the unindented version, but I think  I prefer this one.